ada_2_fhir-r4/zibs2020/ada_project_drivers/_driverInclude.xsl

```diff
@@ -256,6 +256,9 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                     <xsl:with-param name="subject" select="$subject"/>
                 </xsl:apply-templates>
             </xsl:when>
+            <xsl:when test="$localName = 'participatie_in_maatschappij'">
+                <xsl:apply-templates select="$in" mode="nl-core-ParticipationInSociety"/>
+            </xsl:when>
```

Comment:
The following line should be added: `<xsl:with-param name="subject" select="$subject"/>`
Otherwise the patient will not be added to the FHIR instance as subject.

Reply:
Added! Good find.